test/functional/vimscript/null_spec.lua

```diff
@@ -1,14 +1,26 @@
 local helpers = require('test.functional.helpers')(after_each)
 
 local curbufmeths = helpers.curbufmeths
-local redir_exec = helpers.redir_exec
 local exc_exec = helpers.exc_exec
 local command = helpers.command
 local clear = helpers.clear
 local meths = helpers.meths
 local funcs = helpers.funcs
 local eq = helpers.eq
 
+local function redir_exec(cmd)
```

Comment:
These tests highligh depend on the quirks of `:redir`, so `redir_exec` survives here.